front-end/src/data/filters.tsx

```diff
@@ -0,0 +1,13 @@
+export const jobType: string[] = [
```

Comment:
make function in service layer and return dummy data from there. function in service layer will call backend APIs when they will be ready to use. This will also help you use proper react hooks or redux functions on frontend.